src/components/jogging/translationHelper.tsx

```diff
@@ -0,0 +1,10 @@
+import { useTranslation } from "react-i18next"
+
+export const getTabTranslation = (tab: string): string => {
+  const { t } = useTranslation()
```

Comment:
If you are using a hook inside a function its common practice to transform the function to a hook. Like: useTabTranslation() which returns the getTabTranslation function as e.g. translate(value: string) -> string

Reply:
usage would be 
```
const { translate } = useTabTranslate()
const tabName = translate("cartesian");
```